doc/man/8/ceph.rst

```diff
@@ -1724,7 +1724,25 @@ Options
 
 .. option:: -f {json,json-pretty,xml,xml-pretty,plain,yaml}, --format
 
-	Format of output. Note: yaml is only valid for orch commands. 
+	Format of output.
+
+    Note: yaml is only valid for orch commands.
+
+.. option:: --daemon-output-file OUTPUT_FILE
+
+    When using --format=json|json-pretty, you may specify a file name on the
```

Comment:
Why do we limit it to json only? Any format should be dumpable locally, shouldn't it?

Reply:
You would need to extend the formatter to write to a file instead of a stringstream. There could even be some refactoring to plug that in naturally. However, that's beyond the scope of this PR.

Reply:
ack